rnFirebaseAuth/components/login.js

```diff
@@ -86,7 +86,7 @@ const styles = StyleSheet.create({
     flexDirection: "column",
     justifyContent: "center",
     padding: 35,
-    backgroundColor: '#fff'
+    backgroundColor: '#FAF9F6'
```

Comment:
want to keep coloring consistent with styling recommendations from client when she gives them